website/intro/getting-started/install.html.markdown

```diff
@@ -57,6 +57,17 @@ If you get an error that `terraform` could not be found, your `PATH` environment
 variable was not set up properly. Please go back and ensure that your `PATH`
 variable contains the directory where Terraform was installed.
 
+## Installing Terraform Providers
+
+Terraform uses a plugin based architecture to support the numerous infrastructure 
+and service providers available. As of Terraform version 0.10.0, each "Provider" is it's 
+own encapsulated binary distributed separately from Terraform itself. The core 
+Terraform binary downloaded and installed above will automatically download and 
+install any Provider binary needed for you, so you don't have to find and install 
+them yourself. A list of currently supported Providers can be found on 
+[our Providers page](/docs/providers/index.html).
```

Comment:
I think I'd remove this whole section and put it as a paragraph in the "Init" section of the "Build Infrastructure" page. I think during installation the user will be blissfully unaware either way of where providers are, and I'd rather keep it simple.

Then, during the "init" section, we can explain that providers are downloaded then, especially since they'll see it in the output.